packages/@yoda/barbapapa/index.js

```diff
@@ -0,0 +1,82 @@
+var fs = require('fs')
+var promisify = require('util').promisify
+var path = require('path')
+
+var _ = require('@yoda/util')._
+
+var readdirAsync = promisify(fs.readdir)
+
+module.exports = class Barbapapa {
```

Comment:
The annotation to document how to use this to write Yoda Component in detail is required.

Reply:
Yes, since this is a draft merely for reviewing. Docs would be included if the PR comes to a stable stage.